src/audio/dai.c

```diff
@@ -557,12 +613,12 @@ static void dai_update_start_position(struct comp_dev *dev)
 }
 
 /* used to pass standard and bespoke command (with data) to component */
-static int dai_comp_trigger(struct comp_dev *dev, int cmd)
+static int dai_comp_trigger_int(struct comp_dev *dev, int cmd)
```

Comment:
what does int stand for here?

Reply:
Internal

Reply:
Might be worth fully spelling it out in the function name.

Reply:
ack, I read it as the type.